docs/content/build-image.md

```diff
@@ -0,0 +1,358 @@
+  exec:
+    command: bash
+    arguments:
+      - -c
+      - echo Goodbye World
+```
+
+This bundle, for example, declares a dependency on a bundle named `mysql`. The CNAB specification doesn't provide a mechanism for handling dependency resolution. Porter addresses this capability by resolving any dependencies at build time, including the contents of each dependency in the invocation image. At runtime the contents of that bundle will therefore be in the bundle and the Porter runtime component can execute them successfully.
+
+```console
+$ porter build
+Copying dependencies ===>
+Copying bundle dependency mysql ===>
+Copying mixins ===>
+Copying mixin helm ===>
+Copying mixin exec ===>
+Copying mixin porter ===>
+
+Generating Dockerfile =======>
+[FROM quay.io/deis/lightweight-docker-go:v0.2.0 FROM debian:stretch COPY cnab/ /cnab/ COPY porter.yaml /cnab/app/porter.yaml CMD ["/cnab/app/run"] COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt RUN apt-get update && \  apt-get install -y curl && \  curl -o helm.tgz https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz && \  tar -xzf helm.tgz && \  mv linux-amd64/helm /usr/local/bin && \  rm helm.tgz RUN helm init --client-only # exec mixin has no buildtime dependencies ]
+
+Writing Dockerfile =======>
+FROM quay.io/deis/lightweight-docker-go:v0.2.0
+FROM debian:stretch
+COPY cnab/ /cnab/
+COPY porter.yaml /cnab/app/porter.yaml
+CMD ["/cnab/app/run"]
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+RUN apt-get update && \
+ apt-get install -y curl && \
+ curl -o helm.tgz https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz && \
+ tar -xzf helm.tgz && \
+ mv linux-amd64/helm /usr/local/bin && \
+ rm helm.tgz
+RUN helm init --client-only
+# exec mixin has no buildtime dependencies
+```
+
+As we can see from this output, Porter found the `mysql` dependency and copied it into our bundle. If we examine the `cnab` directory after this build is complete, it should look like this:
+
+```console
+$ tree cnab/
+cnab/
+└── app
+    ├── bundles
+    │   └── mysql
+    │       └── porter.yaml
+    ├── mixins
+    │   ├── exec
+    │   │   ├── exec
+    │   │   └── exec-runtime
+    │   ├── helm
+    │   │   ├── helm
+    │   │   └── helm-runtime
+    │   └── porter
+    │       ├── porter
+    │       └── porter-runtime
+    ├── porter-runtime
+    └── run
+```
+
+Porter found the `mysql` bundle by first looking in the `./bundles` directory. If nothing was found, it then checked `$PORTER_HOME\bundles` for the `mysql` bundle. In this case, it found the bundle and was able to copy it into the directory. Porter also copied the `helm` mixin into our bundle, despite the `porter.yaml` declaring only the `exec` mixin. It did this because the `mysql` dependency requires the `helm` mixin. As we can also see from the Dockerfile, it included the mixin for the `mysql` before the `exec` mixin.
```

Comment:
I think we need a final paragraph summarizing the state of the invocation image and wrapping things up? Maybe something like 

`In the end, regardless of whether there are dependencies, the result is a single invocation image with all of the necessary pieces (config files, binaries, charts, etc included in the image) which can be executed by any tool that supports the CNAB spec... blah blah`